src/test/ui/rfc-2497-if-let-chains/disallowed-positions.stderr

```diff
@@ -513,33 +513,6 @@ warning: the feature `let_chains` is incomplete and may cause the compiler to cr
 LL | #![feature(let_chains)] // Avoid inflating `.stderr` with overzealous gates in this test.
    |            ^^^^^^^^^^
 
-error[E0658]: `match` is not allowed in a `const`
```

Comment:
`::Let` can result in switching on the discriminant, like or-patterns. Alone however, it is not control flow. That said, we should probably gate `let p = s` here too?